security/advancedtls/advancedtls_test.go

```diff
@@ -679,3 +681,83 @@ func TestOptionsConfig(t *testing.T) {
 		})
 	}
 }
+
+func TestGetCertificateSNI(t *testing.T) {
+	// Load server certificates for setting the serverGetCert callback function.
+	serverPeerCert1, err := tls.LoadX509KeyPair(testdata.Path("server_cert_1.pem"),
```

Comment:
`serverPeerCert1` is really the server's certificate, right? Then renaming it as `serverCert1` sounds more appropriate.

Reply:
There are two types of certificates, peer certificate and trust certificate. We are loading server's peer certificates here, which is why I named it to `serverPeerCert1`. I could change the comment to `"Load server peer certificates for setting the serverGetCert callback function."` if that seems more clear to you!

Reply:
I understand that there are two types of certificates. One establishes your identity (called identityCert or just cert), and the other helps validate the certs provided by the other end (called trustCerts, trustRoots or simply roots). We usually call the other end as the peer. So, for the server, the client is the peer and vice-versa. HTH.

Reply:
I see, that makes sense. Thanks for the clarification!